src/autogluon/bench/eval/hardware_metrics/hardware_metrics.py

```diff
@@ -0,0 +1,247 @@
+import csv
+import logging
+import os
+from datetime import datetime, timedelta
+from typing import List, Optional
+
+import boto3
+import pandas as pd
+import typer
+import yaml
+
+metrics_list = []
+aws_account_id = None
+aws_account_region = None
+
+
+def get_job_ids(config_file: str):
+    """
+    This function returns a list of job IDs of all jobs ran for a benchmark run
+    Parameters
+    ----------
+    config_file: str,
+        Path to config file containing job IDs
+    """
+    job_ids = list(config_file.get("job_configs", {}).keys())
+    return job_ids
+
+
+def get_instance_id(job_id):
```

Comment:
Yeah, I think we could pass in a list of job IDs and get a larger response from cloudwatch. Then we'd have to iterate through the dictionary and get the instance IDs (instead of doing `response["containerInstances"][0]`, we'd do `response["containerInstances"][i]` I believe)

Reply:
Sounds good. I think this can be a follow up PR.